docs/index.html

```diff
@@ -30,18 +15,14 @@ <h1>Precompiled wheels commonly used in Linuxserver's images</h1>
     <a href='/alpine-3.21/'>alpine-3.21</a>
     <br />
 
-    <a href='/alpine-edge/'>alpine-edge</a>
+    <a href='/alpine-3.22/'>alpine-3.21</a>
```

Comment:
This needs to be 3.22

Reply:
I did a find & replace so no idea how that happened.